src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -340,7 +345,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CompileEmccArgs Condition="'$(NativeDebugSymbols)' != 'true'">$(CompileEmccArgs) -O2</CompileEmccArgs>
       <CompileEmccArgs Condition="'$(NativeDebugSymbols)' == 'true'">$(CompileEmccArgs) -g3</CompileEmccArgs>
```

Comment:
The `O2` addition here should presumably be deleted now.

Reply:
yes, indeed.